.gitlab-ci-check-docker-acceptance.yml

```diff
@@ -77,7 +77,7 @@ test:acceptance_tests:
     - test:prepare_acceptance
   before_script:
     - apk add git bash
-    - git clone --depth=1 https://github.com/mendersoftware/integration.git mender-integration
+    - git clone -b ${INTEGRATION_REV:-master} --depth=1 https://github.com/mendersoftware/integration.git mender-integration
```

Comment:
Please document this new variable in the top of this file as optional, something like [this](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-build.yml#L15).

Reply:
Will do!

Reply:
Fixed!